crates/holochain_types/src/dna/dna_manifest/dna_manifest_v1.rs

```diff
@@ -172,9 +174,8 @@ pub struct ZomeManifest {
     #[serde(flatten)]
     pub location: ZomeLocation,
 
-    /// The integrity zomes this zome depends on.
-    /// The order of these must match the order the types
-    /// are used in the zome.
+    /// The integrity zomes this zome depends on, if it's a coordinator zome.
+    /// Currently a coordinator zome should have **at most one dependency**.
```

Comment:
Not sure about duplicating this. I believe it should read something along the lines of:

- integrity zomes should have no dependencies, leave field `null` 
- coordinator zomes may depend on no, or exactly 1 integrity zomes

Reply:
sure, happy to make those changes if it reads better for you.

Reply:
(Re: duplication, I'd rather have the information at hand in the places where it's needed, even if it means a tiny bit of duplication)